components/ui-shell/index.scss

```diff
@@ -378,3 +388,38 @@ $prefix: "bx";
     }
   }
 }
+
+.#{$q}-side-nav-footer {
+  @include carbon--breakpoint-up("md") {
```

Comment:
wondering if this should be "lg" and up?

reason: the mobile menu button is available at a certain breakpoint, and in that screen, I don't see the mobile menu footer.

https://user-images.githubusercontent.com/6276074/189955651-0b5cce3f-13c6-433e-afe9-e4db63c95bc2.mov

Reply:
I was wondering about this also, but stuck to the [original navbar implementation](https://github.com/Qiskit/web-components/blob/3671dd9d8c6f2da8eafeb093b905bf2f48565b43/components/navbar/index.scss#L70-L73)

And also noticed on `qiskit.org` we have the same behavior where smaller (tablet) breakpoint doesn't show the social footer, but the smallest (mobile) breakpoint does.

<img width="1193" alt="Screen Shot 2022-09-13 at 9 41 52 AM" src="https://user-images.githubusercontent.com/8097306/189958936-42475157-58e7-4e89-aa19-9aa942bd02fd.png">
<img width="1192" alt="Screen Shot 2022-09-13 at 9 42 00 AM" src="https://user-images.githubusercontent.com/8097306/189958937-0e336058-a158-4ed1-af09-b50e7b9f090d.png">


Do you think we should change this to show social links on the "tablet" view also?

Reply:
IMO, it's an improvement to the current implementation 

but I won't let it block the PR

Reply:
@JRussellHuffman maybe you have an opinion on this ☝🏼

Reply:
Resolving this conversation for now -- if we need to make updates, at least we're aware of where this is...